.gitignore

```diff
@@ -20,3 +20,5 @@
 npm-debug.log
 yarn-error.log
 ###< symfony/webpack-encore-bundle ###
+cghooks.lock
```

Comment:
it should not serve if you add `"cghooks add --ignore-lock"` in `post-install-cmd`
I'm wrong?

Reply:
what do you mean? `--ignore-lock` adds it to the `.gitignore` as it is right now